bindings/rust/s2n-tls/src/raw/connection.rs

```diff
@@ -29,7 +32,9 @@ impl fmt::Debug for Connection {
     }
 }
 
-/// Safety: s2n_connection objects can be sent across threads
+/// # Safety
+///
+/// Callers must ensure access to Connection is thread safe
```

Comment:
This isn't the correct comment to make. Connections are not thread safe but they can be sent to other threads.